Makefile

```diff
@@ -22,8 +23,19 @@ vm-sanity-checks:
 platform-sanity-checks:
 	cd platform_sanity_checks && ${MAKE} ${PASS_DOWN_ARGS}
 
+rmsr/rmsr.ko:
+ifeq ($(shell uname -s),Linux)
+	# Allow git to fail if rmsr has already been cloned.
+	${GIT} clone https://github.com/softdevteam/rmsr.git | true
```

Comment:
We can't do this: it will also fail if you run `git` and e.g. there isn't a network connection.

Doing this stuff in a makefile is always a mess, because the syntax is so awful for it. It would be easier to put this into build.sh, and then we have easier access to shell commands, and we can say things like "if the rsmr directory doesn't exist, clone it".